portal-web/test/functional/com/liferay/portalweb/macros/CookiePreferenceHandlingAdmin.macro

```diff
@@ -54,6 +84,31 @@ definition {
 		CookiePreferenceHandlingAdmin.saveConfiguration();
 	}
 
+	@description = "Enable Cookie Preference handling with Explicit cookie setting disabled"
```

Comment:
Extend description to indicate instance level

Reply:
done